src/codeql.ts

```diff
@@ -166,6 +173,27 @@ export async function downloadDatabase(
   }
 }
 
+export type QueryMetadata = Record<string, unknown> & { kind?: string };
```

Comment:
I was trying to mimic actual query metadata, which contains all kinds of stuff. But I guess none of that would make it into the object after JSON parsing? 🤔

Reply:
Removed in https://github.com/github/codeql-variant-analysis-action/pull/848/commits/5a7856c76de1e019bdbb8098d46bab4e263e128e!

Reply:
It would technically still be there in the object, but the typescript compiler should stop us accessing any of it, so it won't matter if it's there or not. I think it's fine to remove it.